### PR TITLE
Update python-quiz.ts

### PR DESCRIPTION
**Pull request by dhruvjain896**
Added one question

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Added One Question on the concept of python math.trunc in ./src/data/python-quiz.ts

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1531,6 +1531,15 @@ const pythonQuiz = [
     Explanation: "print('%.3f'%x) would round 297.4887 to 3 decimal places.",
     Link: "https://www.freecodecamp.org/news/2f-in-python-what-does-it-mean/",
   },
+   {
+    Question: "What is the result of math.trunc(3.1)?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1535]:
```suggestion
    Question: "In Python, what is the result for math.trunc(3.1)?",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1531,6 +1531,15 @@ const pythonQuiz = [
...
+    Answer: "3",
+    Distractor1: "3.0",
+    Distractor2: "3",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1538]: distractors should be for wrong answers only

```suggestion
    Distractor2: "3.14",
```

File: src/data/python-quiz.ts [hunk 3]
```diff
@@ -1531,6 +1531,15 @@ const pythonQuiz = [
     Explanation: "print('%.3f'%x) would round 297.4887 to 3 decimal places.",
     Link: "https://www.freecodecamp.org/news/2f-in-python-what-does-it-mean/",
   },
+   {
+    Question: "What is the result of math.trunc(3.1)?",
+    Answer: "3",
+    Distractor1: "3.0",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1537]:
```suggestion
    Distractor1: "3.5",
```

File: src/data/python-quiz.ts [hunk 4]
```diff
@@ -1531,6 +1531,15 @@ const pythonQuiz = [
...
+    Distractor2: "3",
+    Distractor3: "0.1",
+    Explanation: "The integral part of the floating point number is returned.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1540]:
```suggestion
    Explanation: "In Python, math.trunc() will return the truncated integer part of the number. When the number is positive, math.trunc() is similar to the .floor() method. But if the number is negative, math.trunc() is similar to the .ceil() method",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by dhruvjain896** [comment]
Hi @jdwilkin4 I committed the requested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**